webrtc.html

```diff
@@ -853,6 +853,12 @@
                   </li>
 
                   <li>
+                    <p>If the supplied description matches the state of all
+                    tracks, clear the negotiation-needed flag.
```

Comment:
Probably needs a more rigorous definition here.  I.e., If the description includes media sections for all output tracks, and an application media section if data channels have been opened.

Question regarding renego: if a track is negotiated off in one round, but not subsequently removed, is there any expectation that a new offer will attempt to add it each time?